public/app/features/search/state/SearchStateManager.ts

```diff
@@ -260,13 +264,14 @@ export function getSearchStateManager() {
   if (!stateManager) {
     const selectedLayout = localStorage.getItem(SEARCH_SELECTED_LAYOUT) as SearchLayout;
     const layout = selectedLayout ?? initialState.layout;
+    const sort = localStorage.getItem(SEARCH_SELECTED_SORT) ?? undefined;
```

Comment:
is there any benefit to the nullish coalescing here? i thought anything set in `localStorage` could only be either a string or `undefined` anyway 🤔

Reply:
`getItem(key: string): string | null;`